core/domain/topic_services.py

```diff
@@ -201,6 +201,24 @@ def get_topic_by_id(topic_id, strict=True, version=None):
             return None
 
 
+def get_topic_by_name(topic_name):
+    """Returns a domain object representing a topic.
+
+    Args:
+        topic_name: str. The name of the topic.
+
+    Returns:
+        Topic or None. The domain object representing a topic with the
+        given id, or None if it does not exist.
+    """
+    topic_model = topic_models.TopicModel.get_by_name(topic_name)[0]
```

Comment:
Why [0]? Is a list returned?

Reply:
Yes a list a returned and 0 because there will always be just one element.